gc/base/standard/ParallelGlobalGC.hpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 1991, 2019 IBM Corp. and others
+ * Copyright (c) 1991, 2020 IBM Corp. and others
```

Comment:
This file has not been changed so copyright should not be updated

Reply:
Good catch. It was modified, at some point, but than I removed those changes. Fixing now.